### PR TITLE
Update TensorNet-QM9.yaml

### PR DESCRIPTION
**Pull request by guillemsimeon**
I updated the optimal hyperparameters for TensorNet on QM9 to obtain the results of the camera-ready paper, which give an improvement over previous accuracy.

### CODE REVIEW

**Comment by RaulPPelaez** [comment]
You changed the seed, but I guess this is just a side effect, right? Like it should not be changing the accuracy noticeably. 
If this is the seed you used for the paper then its ok for reproducibility.

**Comment by guillemsimeon** [comment]
Yes, the seed for the paper, the learning rate, and the batch size.

**Review by RaulPPelaez — APPROVED** [review]
